cmd/harbor/root/member/delete.go

```diff
@@ -0,0 +1,90 @@
+package member
+
+import (
+	"strconv"
+	"sync"
+
+	"github.com/goharbor/harbor-cli/pkg/api"
+	"github.com/goharbor/harbor-cli/pkg/prompt"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// Deletes the member of the given project and Member
+func DeleteMemberCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "delete [projectName or ID] [memberID]",
+		Short:   "delete member by id",
+		Long:    "delete members in a project by MemberID",
+		Example: "  harbor member delete my-project 2",
+		Args:    cobra.MinimumNArgs(0),
+		Run: func(cmd *cobra.Command, args []string) {
+			var wg sync.WaitGroup
+			errChan := make(
+				chan error,
+				len(args),
+			) // Channel to collect errors
+
+			var memberID []int64
+
+			for i, mid := range args {
+				if i != 0 {
+					mID, _ := strconv.Atoi(mid)
+					memberID = append(memberID, int64(mID))
+				}
+			}
+
+			if len(args) > 1 {
+				if args[1] == "%" {
+					api.DeleteAllMember(args[0])
```

Comment:
add a cmd flag to proceed with delete all. like `-a`

Reply:
Yeah, that sounds great. Will add it